src/components/dontneed/Dontneed.module.scss

```diff
@@ -0,0 +1,263 @@
+@import '/src/styles/common.scss';
+
+@mixin mobile {
+  @media (max-width: 786px) {
+    @content;
+    flex-direction: column;
+  }
+}
+
+.careWrapper {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  padding: 100px 50px;
+  width: 100vw;
+  font-family: 'Proxima Nova Alt Lt';
+
+  @include mobile() {
+    padding: 50px 35px;
+  }
+}
+
+.right {
+  width: 50%;
+  position: relative;
+  padding: 0 70px;
+  height: 100%;
+
+  @include mobile() {
+    width: 100%;
+    padding: 0;
+  }
+
+  h2 {
+    font-size: 46px;
+    line-height: 63px;
+    margin-bottom: 25px;
+    color: $blue;
+
+    @include mobile() {
+      font-size: 32px;
+    }
+  }
+
+  ul {
+    list-style-type: none;
+    list-style-position: outside;
+    padding-left: 40px;
+
+    @include mobile() {
+      padding-left: 40px;
+    }
+
+    li {
+      padding-bottom: 30px;
+      padding-right: 50px;
+      font-size: 20px;
+      line-height: 27px;
+      color: $blue;
+      position: relative;
+
+      @include mobile() {
+        font-size: 16px;
+        line-height: 16px;
+        padding-right: 20px;
+      }
+
+      &:before {
+        content: '';
+        left: -40px;
+        top: 5px;
+        position: absolute;
+        @include mobile() {
+          top: 0;
+        }
+      }
+
+      &:nth-child(1)::before {
+        width: 13px;
+        height: 13px;
+        background: #c2e0f6;
+        border-radius: 0 100px 0 0;
+        -moz-border-radius: 0 100px 0 0;
+        -webkit-border-radius: 0 100px 0 0;
+      }
+
+      &:nth-child(2)::before {
+        width: 13px;
+        height: 13px;
+        background: #f6c2e8;
+        border-radius: 0 100px 0 0;
+        -moz-border-radius: 0 100px 0 0;
```

Comment:
i believe the shareable code can be shorten with @extend or @mixin-@include pair